.gitignore

```diff
@@ -3,4 +3,5 @@ composer.lock
 coverage.xml
 coverage/
 node_modules
-dist
\ No newline at end of file
+dist
+thumb_*
```

Comment:
This thumb_* gitignore rule is too broad. I'd suggest changing it to something like `tests/src/thumb_*`